frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
```

Comment:
nit: filenames of `ProjectAccess` and `ProjectAccessForm` can be switched since essentially this is the actual form component. This is how specific import forms are named as well.